Tests/Fixtures/Query/FooWalker.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Query;
+
+use Doctrine\ORM\Query\SqlWalker;
+
+class FooWalker extends SqlWalker
+{
+    public function walkOrderByClause($orderByClause)
+    {
+        $sql = parent::walkOrderByClause($orderByClause);
+
+        return str_replace(' ASC', ' DESC', $sql);
```

Comment:
`return str_replace(' ASC', ' DESC', parent::walkOrderByClause($orderByClause));` is more concise.